### ISSUE TITLE
Make from/living location fields store actual geo info

### ISSUE THREAD

**Issue by simison** [Trustroots/trustroots#395]
We'll need coordinates.

That said, I think it'd be great to let people fill in freetext like "Mars".

Related to #359

**Comment by mrkvon** [comment]
When filling the names of places to `From` and `Location` fields, the [api.mapbox.com](https://api.mapbox.com/geocoding/v5/mapbox.places/Vaasa.json?access_token=pk.eyJ1IjoidHJ1c3Ryb290cyIsImEiOiJVWFFGa19BIn0.4e59q4-7e8yvgvcd1jzF4g&types=country,region,place,locality,neighborhood) already provides coordinates and everything. So it seems to be a question of storing these data in database. (and maybe some script to update the locations saved earlier).

And if the user doesn't select the place from the provided list, then there can be a little warning for user and storing a boolean indicating that the name is not an actual geolocation (so it would be treated accordingly).

**Comment by simison** [comment]
Yep or just `FromLabel` (String) and `From` (Object) to keep it simple. Mongo would even support mixing String/Object but that's probably not smart.

**Comment by simison** [comment]
Noting here that this is more about the flow/UI than it's about technical solution. As @mrkvon pointed out above, the technical side of this is fairly straightforward (and could indeed be the first step).

**Comment by Regenwasser** [comment]
So, if some wireframes are needed here to bring this further I think we should have a chat or call about it. (This seems to be connected to other location features, etc.).

> and could indeed be the first step 

Yes, good idea! If it's fast and easy, better do this first to get some users onto the map.

Maybe the first step could include: 
- make the text run through some geolocator (filter out all countries and things that weren't found)
- add a higher randomisation, so it's not just clustered around city centers.
- change field to Autocomplete addresses and places with something like [this](https://developers.google.com/maps/documentation/javascript/places-autocomplete). Right now only cities are auto-completed. Streets are not recognised, so I assume not many entered them).
